<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWindow</class>
 <widget class="QMainWindow" name="registerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>150</width>
      <height>92</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>211</width>
      <height>134</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>账号🙉（必填）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerIDInputLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>329</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>211</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码🙈（必填）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerPasswordInputLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>329</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>141</width>
      <height>276</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册身份</string>
    </property>
   </widget>
   <widget class="QComboBox" name="registerLevelSelectComboBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>普通用户🥳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>商家🥸</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>食堂管理员🤠</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="registerConfirmButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>329</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerAddressInputLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>329</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>111</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>地址🧭</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerNicknameInputLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>329</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>111</width>
      <height>134</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>名称👾</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>271</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>电话号码📞（必填）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerTelephoneInputLine">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>329</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
